src/core/opamStd.mli

```diff
@@ -413,6 +410,30 @@ module Env : sig
 
   (** Utility function for PowerShell strings. *)
   val escape_powershell: string -> string
+
+  (** {3 Environment variable handling} *)
+
+  (** Environment variable names *)
```

Comment:
worth add some documentation to explain the purpose of the module, ie the need for windows to have specific comparison

Reply:
Done!